### DISCUSSION TITLE
Wifi Probleme - mit dem Fritz Repeater verbinden anstatt mit dem Router

### DISCUSSION THREAD

**Discussion by pimp1310** [jomjol/AI-on-the-edge-device#2289] [Q&A] [answered]
Hallo,

mein Problem ist das seit gestern sich mein Wasserzähler und Stromzähler anstatt mit dem 30cm entferntem Repeater (Wlan-Mesh) lieber mit dem Router verbindet, nur steht dieser ganz oben und dadurch habe ich ein sehr schlechtes Signal, so schlecht das stellenweise die UI nicht aufgeht.

WIFI Signal: Weak (-82dBm)
WIFI Signal: Unreliable (-86dBm)

ist es möglich dem ESP zu sagen das er sich nur auf eine bestimmte mac verbinden darf? dann könnte ich die mac Adresse des Repeaters angeben.

sowas in der art

;++++++++++++++++++++++++++++++++++
; AI on the edge - WLAN configuration
;++++++++++++++++++++++++++++++++++
; ssid: Name of WLAN network (mandatory), e.g. "WLAN-SSID"
; password: Password of WLAN network (mandatory), e.g. "PASSWORD"

ssid = "UPC5518865"
password = "nsjY6hzxnev8665"
mac = "XXXXXXXXXX"

ich wüsste nicht wie ich das sonst hinkriegen soll.
danke

**Comment by caco3** [comment, marked as answer]
Can you try to set [RSSI Threshold](https://jomjol.github.io/AI-on-the-edge-device-docs/Parameters/#parameter-rssithreshold) and see if that helps?

Also you did not state which version you use!

**Reply by pimp1310** [reply]
klar kann ich probieren, 0 bis -100 ist der Bereich richtig? dann müsste ich bei einem aktuellen wert von  (-83dBm) bei RSSIThreshold -70 eintragen als sicherheit?

**Reply by pimp1310** [reply]
okay das klappt wirklich perfekt nun sieht es wesentlich besser aus

WIFI Signal: Excellent (-53dBm)

**Comment by ghost** [comment]
Der ESP32 kennt nur das 2,4 GHz Band. Insofern könntest du prüfen, ob das auch in deinem Mesh-Repeater eingestellt ist. Das mit der MAC ist für mich das bekämpfen von Symptomen.

**Reply by pimp1310** [reply]
ja habe 2.4GHZ und 5Ghz getrennt und aktiviert, der esp war ja mal damit verbunden, aber nun will er nicht mehr.
ich finde nicht das das bekämpfen von Symptomen ist sondern eine zusätzliche möglichkeit zu steuern mit wem er sich verbindet, vorallem bei Mesh sehr wichtig, da er anscheinend das bestreben hat sich immer mit dem router zu verbinden anstatt mit dem repeater egal wie schwach das signal ist.

**Comment by caco3** [comment]
> password = "nsjY6hzxnev8"

Falls das dein richtiges Passwort ist, solltest Du dir jetzt ein Neues suchen!

**Reply by pimp1310** [reply]
ne ist es nicht, aber dennoch danke für den Hinweis :-)
